backend/controllers/contributions.js

```diff
@@ -15,13 +21,19 @@ const AuthContribution = {
   async verify(req, res) {
     const verifyIdentity = await AuthContribution.verifyIdentity(req.body.token)
     if (verifyIdentity.status !== 200)
-      return res.send({
-        msg: "Not found",
-        statusCode: 404,
-      })
+      return AuthContribution.notAuthorized(res)
     res.cookie("contribution_token", req.body.token)
     return res.redirect(301, "/")
   },
+  isLogged(req, res, next) {
+    AuthContribution.verifyIdentity(req.body.token).then((verifyIdentity) => {
```

Comment:
You could put the all verifyIdentity in your `notAuthorized`method and return a boolean to use it in both `verify`and `isLogged`
And it's a good practice to decide between using await or then